static/app/gettingStartedDocs/react-native/react-native.tsx

```diff
@@ -72,6 +74,27 @@ shopCheckout() {
   }
 }`;
 
+const getReplaySetupSnippet = (params: Params) => `
+import * as Sentry from '@sentry/react-native';
+
+Sentry.init({
+  dsn = "${params.dsn.public}",
```

Comment:
```suggestion
  dsn: "${params.dsn.public}",
```

Reply:
oh if this is the right syntax we should also update our docs, it's wrong there too https://docs.sentry.io/platforms/react-native/session-replay/

cc @krystofwoldrich

Reply:
Yes, this was an oversight in the docs, thank you for fixing it there as well.